### ISSUE TITLE
Add integration tests

### ISSUE THREAD

**Issue by jeddy3** [stylelint/stylelint#202]
[/src/\__tests\__/index.js](https://github.com/stylelint/stylelint/blob/master/src/__tests__/index.js) contains:

```js
// @todo minimal integration tests
```

@MoOx just mentioned it over in #15 about the need for some more tests before we release `0.1.0`. 

Is this where those tests are needed? 

I'll need guidance with this as it's a little out of my comfort zone. What type of tests should go here and how should they function? Is there something either of you could put down first, so I can follow your lead on this?

**Comment by MoOx** [comment]
For now we should add minimal testing that stylelint can be correctly executed as a postcss plugin.
Probably something like this: 

https://github.com/cssnext/cssnext/blob/bcdb1cdeb0a46d88d960e9a773ae5b1af6677844/src/__tests__/api.js#L38-L42

with tests on the content of postcss `result.warnings()`.

We need to test like one rule only in a minimal postcss workflow. We can add more tests later if we add a nother api than just a usage as a postcss plugin.

**Comment by jeddy3** [comment]
Done in #225
